scripts/rollup/build.js

```diff
@@ -150,6 +150,12 @@ function updateBundleConfig(config, filename, format, bundleType, hasteName) {
   });
 }
 
+function setUseFiberEnvVariable(useFiber) {
+  return {
+    'process.env.ROLLUP_USE_FIBER': useFiber,
```

Comment:
Hmm, I don't understand. They would still be specified at the build time, just without a need to look at `process`.